src/otx/v2/adapters/torch/mmengine/mmaction/engine.py

```diff
@@ -0,0 +1,293 @@
+"""OTX adapters.torch.mmengine.mmaction.Engine API."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import copy
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import torch
+from mmaction.registry import VISUALIZERS
+
+from otx.v2.adapters.torch.mmengine.engine import MMXEngine
+from otx.v2.adapters.torch.mmengine.mmaction.registry import MMActionRegistry
+from otx.v2.adapters.torch.mmengine.mmdeploy import AVAILABLE
```

Comment:
Quite misleading when I see where it is referenced. How about this?
```suggestion
from otx.v2.adapters.torch.mmengine.mmdeploy import AVAILABLE as IS_MMDEPLOY_AVAILABLE
```

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2627/commits/428a39c498376a7852400c5618d8c9d6cca72810